### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by cyberW0MBAT**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,47 +15,56 @@
...
 	  <a class="downloadButton" href="#">Download CV</a>
 	</nav>
 
 	  <header>
 	    <div>
-	      <h1 id="whiteText" class="paddingBottom">Hello, I'm Genadi Kozarev.</h1>
-		  <h1 id="whiteText" class="paddingBottom">I'm a full-stack software developer.</h1>
+	      <h1 class="whiteText" class="paddingBottom">Hello, I'm Genadi Kozarev.</h1>
+		  <h1 class="whiteText" class="paddingBottom">I'm a full-stack software developer.</h1>
 	      <button class="portfolioButton">Portfolio</button>
```
Comment (Albertoroldanq) [inline index.html:28]: this button doesn't do anything, probably you would like it jump to the portfolio section? I think you can do it with a href="#portfolio section ID" --> you will need to set an ID to the portfolio section

File: index.html [hunk 2]
```diff
@@ -15,47 +15,56 @@
...
 	    </div>
 	  </header>
```
Comment (Albertoroldanq) [inline index.html:31]: could add `<main>` tag to separate header and footer, I think is more semantic but is up to you, just a suggestion

File: style.css [hunk 3]
```diff
@@ -7,6 +7,10 @@ html {
   font-family: 'Arya', sans-serif;
 }
 
+body {
+	background-color: #00BBF9;
```
Comment (Albertoroldanq) [inline style.css:11]: too much indentation maybe?

File: style.css [hunk 4]
```diff
@@ -77,49 +81,56 @@ header {
...
-    align-items: center;
-}
-
 h2 {
 	font-size: 2rem;
 }
 
-.descriptionAboutMe {
+.blueBackground {
+	background-color: #00BBF9;
```
Comment (Albertoroldanq) [inline style.css:89]: indentation

File: style.css [hunk 5]
```diff
@@ -77,49 +81,56 @@ header {
...
+}
+
+h2.whiteText {
+	margin-left: 25%;
```
Comment (Albertoroldanq) [inline style.css:93]: indentation

File: index.html (hunk 2 above)
Comment (cyberW0MBAT) [reply index.html:31]: Cheers

File: style.css (hunk 3 above)
Comment (cyberW0MBAT) [reply style.css:11]: sharp eye! thanks

**Review by Albertoroldanq — CHANGES_REQUESTED** [review]
Hey Genadi, great job! you code looks awesome, clear and easy to understand. i just added some minor details. 
- I have commented a couple of lines mentioning the indentation, but I have seen there are two kind of indentations and I didn't not think it was necessary to comment in all the lines, maybe you could choose one of both ( to keep consistency in your code).
- I have noticed you haven't added any media query, although your website looks pretty responsive without it ( I might suggest to add a media query to reduce the space on the sides, probably with just changing the width value of the selector `div.innerContainerAboutMe, div.projectContainer`
For the rest of you code, well done, looks great for me!

File: index.html [hunk 6]
```diff
@@ -15,47 +15,56 @@
...
 	  </div>
 	  <a class="downloadButton" href="#">Download CV</a>
 	</nav>
 
 	  <header>
 	    <div>
-	      <h1 id="whiteText" class="paddingBottom">Hello, I'm Genadi Kozarev.</h1>
-		  <h1 id="whiteText" class="paddingBottom">I'm a full-stack software developer.</h1>
+	      <h1 class="whiteText" class="paddingBottom">Hello, I'm Genadi Kozarev.</h1>
+		  <h1 class="whiteText" class="paddingBottom">I'm a full-stack software developer.</h1>
```
Comment (Albertoroldanq) [inline index.html:27]: I missed this, I am not sure it's correct to use two h1 tags. If you want to wrap all this content in a `<h1>` maybe you can try with a `<span>` inside the `h1` and set the `span` with `display:block`

Comment (cyberW0MBAT) [reply index.html:27]: Yeah, good shout. It will look more professional 100%

**Pull request merged**